pkg/cedar/intake/schemas/easi_system_intake.json

```diff
@@ -54,152 +54,336 @@
       ],
       "properties": {
         "adminLead": {
-          "type": "string"
+          "type": "string",
+          "description": "Government Admin responsible for handling request",
+          "examples": [
+            "John Doe"
+          ]
         },
         "archivedAt": {
-          "type": "string"
+          "type": "string",
+          "description": "Timestamp of when request was archived",
+          "examples": [
+            "2022-02-17 14:34:43.405934+00"
+          ]
         },
         "businessNeed": {
-          "type": "string"
+          "type": "string",
+          "description": "Business Need (problem space",
```

Comment:
Is this description just missing a closing paren, or should there be more text there?